src/containers/questions/questionListContainer.js

```diff
@@ -0,0 +1,14 @@
+import { connect } from "react-redux";
+import questionsList from "../../components/questions/questionsList";
+import { getProductQuestions } from "../../redux/actions/questionsActions/getQuestions";
+
+const mapStateToProps = (state, props)=>({store:state});
```

Comment:
Did you plan on using props here? Interested in what you would be passing down to question list from above.

Reply:
Honestly i just added that when i was was looking through YOUR code to remind myself that it was an option! Thats definitely something that might slip my mind at a later time but be handy to remember. Will probably remove it at some point once all my redux set up is finalized